polybar-scripts/player-cmus/player-cmus.sh

```diff
@@ -1,16 +1,26 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Could you use `/bin/sh` and fix all `shellsheck` errors? I don't see why bash is explicitly needed here.

Reply:
Sure, will do!